src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -155,34 +166,151 @@ export function EmbeddedPositionSearchPanel({
         ? <WrapperComponent {...additionalProps}>{internals}</WrapperComponent>
         : internals;
 
+    const keyVal = (k, v, isLast) => (
+        <>
+            <Typography component='span' color={'primary'}>{k}: </Typography> {v}
+            {!isLast && ', '}
+        </>
+    );
+
+    function SearchSummary({title, searchType, coords, radius, fileName, rows, showCoords, showRadius, showFileInfo, sx}) {
+
+        //Label/Key & Value pairs do display, calculating here to determine easily where the last comma should be
+        const keyValuePairs = [
+            { k: 'Search Type', v: searchType },
+            ...(showRadius && searchType === 'Cone' ? [{ k: 'Search Radius', v: radius }] : []),
+            ...(showCoords && searchType !== 'Multi-Object' && coords ? [{ k: 'Coordinates', v: coords }] : []),
+            ...(showFileInfo && searchType === 'Multi-Object' && fileName && rows ? [
+                { k: 'File Name', v: fileName },
+                { k: 'Rows', v: rows }
+            ] : [])
+        ];
+
+        return (
+            <Stack sx={sx}>
+                {title && <><Typography color={'warning'} level='title'>{title}</Typography></>}
+                <Typography color={'neutral'} level='body-md'>
+                    {keyValuePairs.map((pair, index) =>
+                        keyVal(pair.k, pair.v, index === keyValuePairs.length - 1)
+                    )}
+                </Typography>
+            </Stack>
+        );
+    }
+
+    const Header = function(isOpen) {
+        const {groupKey} = useContext(FieldGroupCtx);
+        const reqObj = getFieldGroupResults(groupKey,true);
+
+        const [getTargetWp,]= useFieldGroupValue(targetKey);
+        const [getHiPSRadius,]= useFieldGroupValue(sizeKey);
+        const [getPolygon,]= useFieldGroupValue(polygonKey);
+
+        const userEnterWorldPt= () =>  parseWorldPt(getTargetWp());
+        const worldPt = formatWorldPtToString(userEnterWorldPt());
+
+        const searchType = doGetConeAreaOp() === CONE_CHOICE_KEY ? 'Cone' : (doGetConeAreaOp() === POLY_CHOICE_KEY  ? 'Polygon' : 'Multi-Object');
+        const coords = searchType === 'Cone' ? worldPt : getPolygon();
+        const radius = getHiPSRadius();
+
+        //in case of Multi-Object, get the fileName & rows
+        const fileName = searchType === 'Multi-Object' ? reqObj?.uploadInfo?.fileName : undefined;
+        const rows = searchType === 'Multi-Object' ? reqObj?.uploadInfo?.totalRows : undefined;
+
+        const defSlotProps = {
+            searchSummary: {
+                title: undefined,
+                showCoords: false,
+                showRadius: false,
+                showFileInfo: false,
+                searchType, coords, radius, fileName, rows,
+                sx:{}
+            },
+        };
```

Comment:
#### All this work should be done in `SearchSummary`
- for force a rerender use:
```js
useFieldGroupRerender([targetKey,sizeKey,polygonKey,CONE_AREA_KEY]);

```

#### `SearchSummary` should only take a request

```jsx
 <Slot component={SearchSummary} request={reqObj}/>
```

   - `SearchSummary` is just a default implementation that will probably always be replaced. We may may need to make a helper components for replacing it but that will be in future steps.

   - `SearchSummary`'s job is to take a request object and make the header UI.